packages/framework/build.xml

```diff
@@ -488,6 +629,17 @@
         </exec>
     </target>
 
+    <target name="environment-load" description="Load Symfony environment into Phing properties." hidden="true">
+        <if>
+            <not><isset property="environment"/></not>
+            <then>
+                <exec command="php bin/console about | grep Environment | sed 's/Environment//;s/ //g'" outputProperty="environment"/>
```

Comment:
I do not understand `sed` enough to be able to validate this, however, it works, so ok :grimacing: